pkgs/test/cc-wrapper/default.nix

```diff
@@ -46,6 +46,17 @@ in stdenv.mkDerivation {
       $READELF -d ./atomics.so | grep libatomic.so && echo "ok" >&2 || echo "failed" >&2
     ''}
 
+    # Test that linking libc++ works, and statically.
+    ${lib.optionalString isCxx ''
```

Comment:
Is this the right condition? we need to make sure we're using `libc++` not `libstdc++` or whatever.

Reply:
Good spot, but I think the condition is actually the correct one from how isCxx is currently defined; it actually is asking if it's libcxx. For libstdc++ environments, `stdenv.cc.libcxx` is null:

https://github.com/NixOS/nixpkgs/blob/32f0167ef1422bfee3f1ee4576d5d992a489d24e/pkgs/test/cc-wrapper/default.nix#L12